src/drawing/backend_impl/bitmap.rs

```diff
@@ -4,9 +4,9 @@ use std::marker::PhantomData;
 
 #[cfg(all(not(target_arch = "wasm32"), feature = "image"))]
 mod image_encoding_support {
-    pub(super) use image::{ImageBuffer, ImageError, Rgb};
+    pub(super) use image::{ImageBuffer, ImageError, Pixel};
     pub(super) use std::path::Path;
-    pub(super) type BorrowedImage<'a> = ImageBuffer<Rgb<u8>, &'a mut [u8]>;
+    pub(super) type BorrowedImage<'a, P> = ImageBuffer<P, &'a mut [u8]>;
```

Comment:
Note: bounds on generic parameters are not enforced in type aliases.

https://github.com/rust-lang/rust/issues/21903